### PR TITLE
fix(template): enable race detection in tests only

### PR DESCRIPTION
**Pull request by iawia002**
**What this PR does / why we need it**:

Do not enable the `-race` flag in production, it will affect performance.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/nirvana/project/init.go [hunk 1]
```diff
@@ -836,7 +836,7 @@ $(GOLANGCI_LINT):
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(BIN_DIR) v1.23.6
 
 test:
-	@go test -coverprofile=coverage.out ./...
+	@go test -race -coverprofile=coverage.out ./...
```
Comment (hezhizhen) [inline cmd/nirvana/project/init.go:839]: 把上面 race 的注释移到这一行的上面

Comment (iawia002) [reply cmd/nirvana/project/init.go:839]: Nice catch

**Comment by hezhizhen** [comment]
LGTM

**Pull request merged**